MtfhReportingDataListener.Tests/UseCase/TenureUpdatedUseCaseTests.cs

```diff
@@ -164,12 +160,11 @@ public void BuildTenureRecordCanSetOneStringValueToAGenericRecord()
                 ]
             }";
 
-            var receivedRecord = _sut.BuildTenureRecord(schema, _tenure);
+            var receivedRecord = ExecuteBuildTenureRecord(schema, _tenure);
```

Comment:
Why are we no longer using the tenureRecord that is being built from the usecase?

Reply:
I don't think I understand your question here? This is in the use case tests, the method takes the fake tenure we are making, adds some metadata around it so it has the correct structure then calls the method that is being tested. I extracted it into a method so that we didn't have to add the metadata to each test where we are writing out a schema.

Reply:
I've just seen the code that I missed out on. Are they any tests to check the BuildTenureRecord is working as expected (with the changes you've made if you're going to use a fake tenure it doesn't necessarily confirm that the buildTenureRecord() is still working as expected unless I've missed something out here)?